### PR TITLE
[BZ1836311]: Add a note 

### PR DESCRIPTION
**Pull request by xenolinux**
OCP version: 3.11
[preview](https://deploy-preview-41965--osdocs.netlify.app/openshift-enterprise/latest/admin_guide/manage_nodes.html#deleting-nodes)
QE contact @sunilcio LGTMed
[bug](https://bugzilla.redhat.com/show_bug.cgi?id=1836311)

### CODE REVIEW

File: admin_guide/manage_nodes.adoc [hunk 1]
```diff
@@ -280,7 +280,12 @@ $ oc delete node <node>
 ----
 $ oc get nodes
 ----
-+
+
+[NOTE]
+====
+`$ oc delete node <node>` command does not clean the filesystem of a node. A node can register itself if you do not perform the xref:../install/uninstalling_cluster.adoc#uninstalling-nodes[uninstallation of a node].
```
Comment (ctauchen) [inline admin_guide/manage_nodes.adoc:286]: * s/`$ oc delete node <node>` command/The `$ oc delete node <node>` command
* As a naive reader, it's not clear what it means for a node to "register itself". Do you mean a node that you have performed `$ oc delete node <node>` on? Register itself with OpenShift?
* Consider "if you do not xref ...blabla.adoc[uninstall the node].

Comment (xenolinux) [reply admin_guide/manage_nodes.adoc]: Addressed

File: admin_guide/manage_nodes.adoc [hunk 2]
```diff
@@ -280,7 +280,12 @@ $ oc delete node <node>
 ----
 $ oc get nodes
 ----
-+
+
+[NOTE]
+====
+The `$ oc delete node <node>` command does not clean the filesystem of a node. A node which has been deleted, can register itself with OpenShift if you do not perform the xref:../install/uninstalling_cluster.adoc#uninstalling-nodes[uninstall the node].
```
Comment (ctauchen) [inline admin_guide/manage_nodes.adoc:286]:
```suggestion
The `$ oc delete node <node>` command does not clean the filesystem of a node. A node that has been deleted can register itself again with {product-title} if you do not also xref:../install/uninstalling_cluster.adoc#uninstalling-nodes[uninstall the node].
```

Comment (xenolinux) [reply admin_guide/manage_nodes.adoc]: Addressed

**Pull request closed without merging**